data/strings/strings.txt

```diff
@@ -34118,7 +34456,7 @@ fa = دوباره نپرس
     ja = インターネットに接続して、ホテルを日付で絞り込み検索します。
     ko = 인터넷에 연결해 날짜를 통해서 호텔들을 필터 하세요.
     no = Koble seg til Internett for å velge ankomst- og avreisedato.
-    pl = Połącz się z Internetem aby wybrać datę zameldowania i wymeldowania.
+    pl = Połącz się z Internetem aby wybrać datę zameldowania i wymeldowania.
```

Comment:
Please revert.

Reply:
Nothing just lines changed after translate and add new lines

Reply:
nothing changed i checked again its right